src/metpy/plots/declarative.py

```diff
@@ -1043,9 +1043,12 @@ def _cmap_obj(self):
     def _norm_obj(self):
         """Return the normalization object.
 
-        Converts the tuple image range to a matplotlib normalization instance.
+        If `image_range` is a matplotlib normalization instance, returns it, otherwise converts 
```

Comment:
One small minor linting issue with an extra space at the end of line which is also causing the line to go over the 95 character limit.